src/app.css

```diff
@@ -67,3 +67,19 @@
 body {
     font-family: 'Outfit', sans-serif;
 }
+
+.leftColumn {
```

Comment:
this pattern is highly discouraged by tailwinds; I'm not 100% sold on their arguments but would prefer to stick to their way of doing things since they've given thought to these things for far longer than:

https://tailwindcss.com/docs/reusing-styles#compared-to-css-abstractions

Reply:
Let me think about it. I don't know if I agree with them but... maybe?

Reply:
one part of their argument that resonates with me is that if you need to copy the style too much you should probably just abstract to components

Reply:
I got rid of .leftColumn but left the other two for now. It's not clear to me yet how all the pages will fit together (what has chat columns, what doesn't. I think we'll be able to refactor that in the future and ditch these styles then.